### PR TITLE
Report View | 'get-report' endpoint and updates to component

### PR DESCRIPTION
**Pull request by itskarelleh**
This PR does the following:
- adds a new endpoint(/api/get-report) that fetches and returns report details from the database
- updates and fixes values passed to the ReportView component and its input fields
- page and layout moved from **/reports-view to /reports/[incident_report_id]**
- adds .env.example

### CODE REVIEW

File: .env.example
Comment (johnjang94) [inline .env.example:1]: Would you like to update this file with the following code?

`NODE_ENV="production"` - yes, possible values: development
`TWILIO_ACCOUNT_SID="AC3c345a8fec4d73c5fbf5410f25a80bf5"`
`TWILIO_AUTH_TOKEN="02675c28f60f0acf3de8e909e247d9b6"`
`TWILIO_API_KEY = "SKa70998c952afee7a1c6a00f598319b29"`
`TWILIO_API_SECRET = "T7RmD5YJzDd1gsK0BWa8ynfWT8qcSR3p"`
`TWILIO_SERVICE_SID = "IS4617b24af2ce469d82f851c29ada2ebe"`
`DATABASE_URL="mongodb+srv://gaurav:Ph1llyTruce2024@phillytruce.glcfb6p.mongodb.net/phillytruce_db"`

Comment (itskarelleh) [reply .env.example:1]: additional env vars added ✅

**Comment by divya-prav** [comment]
I would like to understand that the Report object declaration (/components/report-view/index.tsx) didn't include the creator_user_id. Is there any reason for this?
Screenshot 2024-08-30 at 11 17 19 AM

**Comment by itskarelleh** [comment]
> I would like to understand that the Report object declaration (/components/report-view/index.tsx) didn't include the creator_user_id. Is there any reason for this? Screenshot 2024-08-30 at 11 17 19 AM

The creator user is used to display the the name of the SPM.

**Comment by johnjang94** [comment]
Hello Karelle, I have reviewed the issue and came to understand that the entire issue is being caused due to API being non-existent. Having said that, I rather created a mock version and see if we could resolve this conflict for now. Please take a look at how the mock version is created and how it is being handled in the following link:

[Mock Version](https://github.com/johnjang94/phillytruce-frontend/tree/report-test)
This is only my suggestion --- how you implement it is all up to you.

I hope that this helps. Cheers.

**Comment by johnjang94** [comment]
I made a comment on the Slack channel. 
Please refer to the following link: [report-test](https://github.com/johnjang94/phillytruce-frontend/tree/report-test/src) and the comment I made on the thread on Slack for your reference.

With my changes, this is how it looks like:

https://github.com/user-attachments/assets/7904d918-4797-4588-92cc-83211b725a6d

**Pull request merged**